utils/scripts/load-binary.sh

```diff
@@ -165,15 +165,9 @@ if [ "$TARGET" = "java" ]; then
     ../utils/scripts/docker_base_image.sh ghcr.io/datadog/dd-trace-java/dd-trace-java:latest_snapshot .
 
 elif [ "$TARGET" = "dotnet" ]; then
+    assert_version_is_dev
     rm -rf *.tar.gz
-
-    if [ $VERSION = 'dev' ]; then
-        ../utils/scripts/docker_base_image.sh ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest_snapshot .
-    elif [ $VERSION = 'prod' ]; then
-        ../utils/scripts/docker_base_image.sh ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest .
```

Comment:
[Note to reviewers] We haven't been using `ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest`. It was last updated July 2023.